src/base/logger.h

```diff
@@ -52,18 +52,18 @@ namespace Log
 
     struct Msg
     {
-        int id;
         qint64 timestamp;
+        int id;
```

Comment:
I would want to see `id` at the first place. Maybe rearrange it?

Reply:
id, type, timestamp, message

Reply:
I don't mind but I suppose `struct Peer` would need to follow right? Would the following be ok?
`id, blocked, timestamp, ip, reason`

Reply:
>Would the following be ok?

Yes

Reply:
Done.